README.md

```diff
@@ -2,24 +2,250 @@
 
 # PMIx Standard
 
-This repository contains the LaTeX source for the PMIx standard document, and discussions regarding the PMIx standard.
+PMIx is an application programming interface (API) specification to provide
+High Performance Computing (HPC) libraries and programming models with
+portable and well defined access to commonly available
+distributed computing system services.
 
-## Contributing to the PMIx Standard
+## Governance
 
-There are two main ways to contribute to the PMIx Standard: Participating in the discussion, and writing sections of the document.
+The PMIx Standard is governed by an Administrative Steering Committee (ASC) comprised of representatives from a consortium of research, academic, and industrial organizations.The ASC shall consist of one representative from each Member organization, as defined below, and shall perform the following functions:
 
-### Participating in the discussion
+* Define the mission of the PMIx Standard.
+* Plan release timelines for versions of the PMIx Standard based on available resources, target features, and target goals of the Member organizations.
+* Vote on all issues requiring community consensus (e.g., acceptance of proposed modifications to the Standard), one vote per Member organization. In addition to voting on proposed modifications, the ASC can be requested to vote on any issue submitted by a Member for consideration.
```

Comment:
It is a bit unclear what a Member is at this point. I think you mean that a Member is an organization, but it isn't clearly defined up front and then in the second sentence here when I am reading I wonder if a Member can be an individual person. It might be good to have a brief definition section up front for terms like Member, Organization, Participant, etc

Reply:
> The ASC shall consist of one representative from each Member organization, **as defined below**

>  Note: Individuals may also become Members so long as their direct employer is not a current Member of the ASC.

It's a bit of a chicken/egg problem - if you start out by defining Member, then the immediate question is "member of what?". I can try to be a little clearer early on...